src/libawkward/array/ListOffsetArray.cpp

```diff
@@ -1269,7 +1269,7 @@ namespace awkward {
           Index64 adjustednonzero(nonzero.length());
 
           struct Error err =
-            awkward_listoffsetarray_getitem_adjust_offsets_index_64(
+            kernel::listoffsetarray_getitem_adjust_offsets_index<int64_t>(
```

Comment:
This one was missed: it should be untemplated and named `kernel::listoffsetarray_getitem_adjust_offsets_index_64`.